docs/README.md

```diff
@@ -64,6 +64,8 @@ By default the agent will run as root. Below are environment-specific instructio
 * [Running as Non-Root on Kubernetes](KUBERNETES.md#run-as-non-root)
 * [Running as Non-Root on OpenShift](OPENSHIFT.md#run-as-non-root)
 
+If you configure the agent to run as non-root, review the [documentation about enabling "statefulness" for the agent](KUBERNETES.md#enabling-persistent-agent-state).
```

Comment:
`KUBERNETES.md#enabling-persistent-agent-state` points to a location that doesn't exist in the markdown file. Was the idea to also modify the `KUBERNETES.md` file?

Reply:
oh, I see that it relates to #109 , nvm.